java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_2/Robot.java

```diff
@@ -0,0 +1,174 @@
+package com.zoolatech.lecture2.tasks._2;
+
+public class Robot {
+    private final Room room;
+    Direction direction;
```

Comment:
why the direction is not defined as private?

Reply:
Already made Direction variable as private.